src/index.tsx

```diff
@@ -0,0 +1,22 @@
+import { render } from "preact"
+import { SearchPageProvider } from "@nosto/search-js/preact/serp"
+import { initNosto } from "./utils/initNosto"
+import Serp from "./components/Serp"
+import { Search } from "./components/Search"
+import "./styles/index.css"
+import { serpConfig } from "./config"
+
+initNosto()
```

Comment:
can you move this to the dev side logic?
for normal search templates init this should not be needed